src/port1.0/portbuild.tcl

```diff
@@ -181,7 +187,7 @@ proc portbuild::build_getjobsarg {args} {
         return ""
     }
     set jobs [option build.jobs]
-    if {![string is integer -strict $jobs] || $jobs <= 1} {
+    if {![string is integer -strict $jobs] || $jobs < 1} {
         return ""
     }
```

Comment:
This will cause `-j1` to be used for make/ninja/scons if `build.jobs` is 1, which seems fine to me. But actually there should be no circumstance in which `build.jobs` could be < 1, so this could be simplified:
```suggestion
```

Reply:
Well, looking a little further, a user can set `buildmakejobs` in macports.conf. If they set it to `0`, we set it to the number of active processors or `2` if we can't determine it. But this might've been the only code that handles if a user set `buildmakejobs` to a negative integer. There's also the remote possibility that a port or portgroup could have set `build.jobs` to `0` or a negative integer, and it's probably good to handle that here too.

There is still other invalid input that we probably don't handle, such as `buildmakejobs` or `build.jobs` being set to a non-integer number or string.